### ISSUE TITLE
Edit アニメイト

### ISSUE THREAD

**Issue by maro-21** [osmlab/name-suggestion-index#4167]
Animeito in Japan sells mainly manga and anime and we already have a tag for such shops: `shop=anime`.
https://en.wikipedia.org/wiki/Animate_(retailer)

**Comment by bhousel** [comment]
Ah yeah, I saw on the iD repo that you don't like the `shop=hobby` tag, and that you've been editing the wiki to try to get people to stop using it.
https://github.com/openstreetmap/iD/issues/7882

I think it's rude of you to try replacing a tag that mappers in Japan are using.. But at this point, I don't really care so I'll just change all these values so the issue goes away.

**Comment by maro-21** [comment]
> and that you've been editing the wiki to try to get people to stop using it.

No, I didn't.
The only thing I did was expanding the list of suggested replacements:
https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dhobby&diff=2017292&oldid=1806009

Everyone can see how the Wiki paged looked like before my edit:
https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dhobby&action=history
https://wiki.openstreetmap.org/w/index.php?title=Tag:shop%3Dhobby&oldid=1806009
There isn't any suggestion of "stop using", just a message "undocumented tag".

> I think it's rude of you to try replacing a tag that mappers in Japan are using

I just help to find a better tag. Before posting here I spent some time and checked carefully what kind of goods every shop offers. It's possible that people didn't know before about `shop=anime` tag or they didn't know that it can also be used for manga stores.

> you don't like the shop=hobby tag

Yes, that's true, because no one wrote what kind of goods this shop offers.
It's hard to use a tag without documentation because every one can interpret it differently.

Thank you for fixing!
